nvi

```diff
@@ -91,6 +91,7 @@ function install_node() {
   NODE_VERSION=$1
   DOWNLOAD_DIR=$2
   INSTALL_DIR=$3
+  local PACKAGE="node-v${NODE_VERSION}-linux-x64"
```

Comment:
This is unrelated to your changes, but it just struck me that we should either clearly label `nvi` a 64-bit installer or handle this more intelligently.